src/components/VoterStatus.js

```diff
@@ -99,6 +99,7 @@ const WelcomeBanner = ({ onboardingOpen }) => {
         hoverTextColor={theme.text.darker_default}
         activeColor={'grey'}
         onClick={onboardingOpen}
+        disabled={!activeAccount}
```

Comment:
@jparklev If we don't have an active account, I think we don't want people trying to go through onboarding setup. Just want to get a second pair of eyes on this, don't want to accidentally lock anyone out.